deploy/openshift/kiali-configmap.yaml

```diff
@@ -18,3 +18,9 @@ data:
     identity:
       cert_file: /kiali-cert/tls.crt
       private_key_file: /kiali-cert/tls.key
+    blacklist:
+      namespaces:
+      - default
+      - istio-operator
+      - kube.*
+      - openshift.*
```

Comment:
see my original comment - let this file as-is and rely on a default. A user can add this section if they want to change the default blacklist. But if blacklist is not specified, we'll have a build-in default set of blacklist items.

I see in openshift we add "openshift.*" - this is fine to add to the default - for kubernetes environments, it will just be a regex that will never match - no harm, no foul.